build-in-docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+programname=$0
+function usage {
+    echo "usage: $programname -i image -v variant [-b branch]"
+    echo "   -i image    specify image name would you like to build"
+    echo "   -v variant  specify variant of the target platform"
+    echo "   -b branch   specify branch name which you would like to use for sources (optional)"
+    echo "   -h          display help"
+
+
+}
+BRANCH='master'
+while getopts hi:v:b: option; do
```

Comment:
Not sure but it looks like you have missed a ":" ``on hi:v:b:``

Reply:
It's not. ":" is for parameters, which receive argument afterwards. "h" stands for "help"